pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - 'packages/*'
+  - 'packages/xstate-svelte/test'
```

Comment:
I guess this might be an accompanying change to https://github.com/statelyai/xstate/pull/4987/files#r1683562667 but I similarly don't think this should be a workspace. 

Could you confirm that this new setup doesn't work after removing this line and the change from that other `package.json`?

Reply:
I changed the import in `UseActor.svelte` and everything is fine now:

https://github.com/statelyai/xstate/pull/4987/files#diff-a89e578c4c6140c646c35fd9f9b0f217b614fd78861ae130c3326300058c756aL4

Reply:
I removed `xstate-svelte/test/package.json` and set `verbatimModuleSyntax: false` in `xstate-svelte/test/tsconfig.json`. That allows `svelte-check` to pass. Not sure if that's fine or not.